test/models/git/exercise_test.rb

```diff
@@ -6,7 +6,7 @@ class GitExerciseTest < ActiveSupport::TestCase
 
     readme_lines = ["# barfoo", "### Something else", "Here"]
```

Comment:
Note that the mock test data does not match the strings that would have been returned by `lines` but does match those returned by `split("\n")`